examples/gocean/eg6/backends_transform.py

```diff
@@ -44,11 +44,11 @@
 from psyclone.psyir.backend.fortran import FortranWriter
 
 
-def trans(psy):
+def trans(psyir):
     ''' Use the PSyIR back-end to generate PSy-layer target code'''
```

Comment:
Yeah, I am tempted to remove it but it would leave a hole and would need to rename many folders, so I will not do it in this PR. But there are a few examples with old stuff that could be clean up.

For now I did what you suggested.